code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents.dm

```diff
@@ -286,22 +286,18 @@ datum/reagent/water/holywater
 datum/reagent/water/holywater/on_mob_life(var/mob/living/M as mob)
 	if(!data) data = 1
 	data++
-	M.jitteriness = max(M.jitteriness-5,0)
+	M.drunkness += 0.5
```

Comment:
holy water gets people drunk, these effects were copied from ethanol. i just chopped out the pasta.